Experimental failure

otan opened the pull request.

otan committed:
test lint fail message

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/56005)
<!-- Reviewable:end -->

otan committed:
build: add instructions to rectify code that is not regenerated

Make it easier for the developer to know how to fix un-generated code by
specifying the required commands.

Release note: None
